src/log.c

```diff
@@ -124,7 +124,11 @@ static int init_logging_session(const Client_Config *c_config, const char *name,
     return 0;
 }
 
-#define LOG_FLUSH_LIMIT 1  /* limits calls to fflush to a max of one per LOG_FLUSH_LIMIT seconds */
+/* limits calls to fflush to a max of one per LOG_FLUSH_LIMIT seconds */
+#define LOG_FLUSH_LIMIT 1
+
+/* Maximum amount of data we can write to a log in a given session */
```

Comment:
This is approximate. If this triggers, it'll usually be a little more than the number below, because we only check when writing the next line, after having exceeded this limit.

Reply:
Done.